project/plugins.sbt

```diff
@@ -1,4 +1,4 @@
-addSbtPlugin("com.jsuereth" % "sbt-pgp" % "2.0.1")
+addSbtPlugin("com.github.sbt" % "sbt-pgp" % "2.1.2")
```

Comment:
Before bumping the library I kept seeing a `file not found` error when attempting to sign artefacts. We can see from [here](https://github.com/sbt/sbt-pgp/releases) that a min of 2.1.1 might be necessary for enabling sbt-pgp to run successfully in CI?

Given it's not a major change, I'd hope there won't be breaking changes, but probably we'd want to test the existing (manual) release process isn't impacted by this.